src/js/Wall.js

```diff
@@ -1,9 +1,9 @@
 import MazeEntity from './MazeEntity.js';
 
 const WallState = {
-    PENDING: 0,
-    CONFIRMED: 1,
-    REMOVED: 2
+    PENDING: 'PENDING',
```

Comment:
for easier debugging

Reply:
I like this - way more clear when inspecting the value!